### PR TITLE
Refactor UserGuide

### PR DESCRIPTION
**Pull request by MechFroG88**
# Changes

1. Add more screenshots
2. Refactor User Guide

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -194,58 +277,79 @@ e.g. To provide a product name of `Nescafe: special edition`, you should type `n
...
+
+<div style="page-break-after: always;"></div>
 
 #### 3.1.3 Updating products : `update`
 
-Updates the product at the specified INDEX.
+Updates the product at the specified `INDEX`.
 
-Format: `update INDEX [TAG:NEW_VALUE ...]`
+Format: `update INDEX [TAG:NEW_VALUE] ...]`
```
Comment (1001mei) [inline docs/UserGuide.md:345]:
```Suggestion
Format: `update INDEX [TAG:NEW_VALUE ...]`
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -534,20 +732,42 @@ Redoes the most recent undone changes made to iBook.
...
 Exits iBook.
 
 Format: `exit`
 
+| Compulsory Fields |  | 
+| Optional Fields   |  |
+
+<div markdown="block" class="alert alert-info">
+
+:information_source: Once exited, the position and the size of ibook will be saved automatically.
```
Comment (1001mei) [inline docs/UserGuide.md:757]: What is the position of iBook?

Comment (MechFroG88) [reply docs/UserGuide.md:757]: I meant to say the position of ibook on the screen.

I will update the sentence for clarity

**Review by mazx4960 — APPROVED** [review]
Looks great!

**Pull request merged**